backend/src/LAN.ts

```diff
@@ -34,6 +42,26 @@ class LAN {
     EventBus.emit(environment.EVENTS.send, environment.frontend)
   }
 
+  async checkOob() {
+    return new Promise<void>((success, failure) => {
+      this.oobAvailable = environment.manufacturerDetails.product.platform === PLATFORM_CODES.REMOTEIT_PI
+      const lan = this
```

Comment:
I think if oob is not available you can just assign oobActive to false and return without the actual wifi. getCurrentConnections check?

Reply:
done!